PcdcAnalysisTools/utils/guppy/guppy.py

```diff
@@ -22,13 +22,17 @@ def downloadDataFromGuppy(path, type, totalCount, fields, filters, sort, accessi
             queryBody["accessibility"] = 'accessible' # accessibility
 
         try:
+            # TODO in the future use the makesignature in the gen3Manager in pcdcutils to make this
             url = path #'http://guppy-service/download'
             headers = {'Content-Type': 'application/json'}
             body = json.dumps(queryBody, separators=(',', ':'))
+            # Use this only for the signature because json.dumps will transform special char in \u encoding which will make a signature javascript won't be able to verify it.
+            # For instance `β` is translated to `\u03b2`
+            body_signature = json.dumps(queryBody, separators=(',', ':'), ensure_ascii=False)
             jwt = get_jwt_from_header()
             headers['Authorization'] = 'bearer ' + jwt
             sm = SignatureManager(key=config["RSA_PRIVATE_KEY"])
-            headers['Signature'] = b'signature ' + sm.sign(body)
+            headers['Signature'] = 'signature ' + sm.sign(body_signature)
```

Comment:
I am not sure what you did here with the PR, but this change is part of another commit and it shouldn't be here.